### DISCUSSION TITLE
Discussions - in Teams or Repo

### DISCUSSION THREAD

**Discussion by ghost** [community/community#44773] [Discussions] [answered]
### Select Topic Area

Question

### Body

I have just created an organization and have several projects with each it's own repo and team.
My question is, why would I use the discussion part from the Teams part of Github, rather than the discussion part from the Repo part - or the other way around.
Any best practices or input on why to use one over the other?

**Comment by jacob-hougaard** [comment, marked as answer]
Github discussions gives us the opportunity to convert discussions in to issues, which can then be put into sprints. so general discussions should maybe not be used here, but discussions about functionality could be useful, as you are able to view a log of what have been agreed upon.
